hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/downloader/block/BlockFileTransformerTest.java

```diff
@@ -109,6 +111,224 @@ void cryptoTransfer(TransferType transferType) {
         });
     }
 
+    @Test
+    void cryptoCreateTransform() {
+        var alias = recordItemBuilder.bytes(20);
+        var expectedRecordItem = recordItemBuilder
+                .cryptoCreate()
+                .record(r -> r.setEvmAddress(alias))
```

Comment:
need to customize transaction body with the evm address set to CryptoCreateTransactionBody.alias

```suggestion
                .transactionBody(b -> b.setAlias(alias))
                .record(r -> r.setEvmAddress(alias))
```